app.py

```diff
@@ -1134,7 +1134,7 @@ def replacesubtree():
 	session.modified = True
 	link = ('<a href="/annotate/accept?%s">accept this tree</a>'
 			% urlencode(dict(sentno=sentno, tree=treeobj.treestr())))
-	return Markup('%s\n\n%s\n\n%s%s\t%s' % (
+	return Markup('%s\t%s\n\n%s%s\t%s' % (
```

Comment:
If editing these `Markup` calls I would change them to use f-strings

Reply:
I take it the tabs are being inserted to demarcate substructure and will later be used for splitting. Would it be better to construct a structured object?